k8s/helmfile/helmfile.yaml

```diff
@@ -47,9 +47,9 @@ helmfiles:
 
 templates:
   default: &default_release
-    missingFileHandler: Warn
+    missingFileHandler: Error
```

Comment:
This requires an empty file to be present for environments that exactly match staging, but I guess that's to be considered a "feature"?

Reply:
I guess so. On the plus side, if someone wants to deviate from the prod config it's already prepared in a way (with the empty file)